app/presenters/export/csv/incoming_trust_presenter_module.rb

```diff
@@ -23,6 +23,16 @@ def incoming_trust_name
     @project.incoming_trust.name
   end
 
+  def incoming_trust_main_contact_name
+    contact = @project.incoming_trust_main_contact_id
+    Contact::Project.find_by(id: contact).name
+  end
+
+  def incoming_trust_main_contact_email
+    contact = @project.incoming_trust_main_contact_id
```

Comment:
Can you add guards to these to handle when 1) the ID is `nil` and 2) the contact is not found